### DISCUSSION TITLE
Printing debug status logs to normal console with Pico Pi from C

### DISCUSSION THREAD

**Discussion by paulhamsh** [micropython/micropython#12898] [Core Development]
Hi
I would like to add some 'debug' status prints to the Pico Pi console from within C (C code in the micropython sources).
Specifically, I am trying to add some statements to the CYW43 driver in the pico-sdk in this file, to show which functions are called and the packet contents ```~/micropython/lib/pico-sdk/src/rp2_common/pico_cyw43_driver/btstack_hci_transport_cyw43.c```
But the request is generic.
On EPS32 a ```printf()``` in the C code shows in the normal console - is that possible with the Pico Pi?

**Comment by russhughes** [comment]
This will printf to the console: 
```mp_printf(&mp_plat_print, "width: %d, height: %d\n", self->width, self->height);```
